conf/muchos.props.example

```diff
@@ -56,6 +56,10 @@ worker_instance_type = %(default_instance_type)s
 #vpc_id = vpc-xxxxx
 # VPC Subnet to launch instances in (optional)
 #subnet_id = subnet-xxxxxx
+# Security group ID to launch in (optional)
+#security_group_id = sg-xxxxxx
+# Associate public IP addresses to non-proxy nodes
+associate_public_ip_to_non_proxy=false
```

Comment:
does this imply the need for public and private subnet config? should this defautll to true?

Reply:
i removed the option.  it should be true until we create options for private & public subnet and test that configuration.